packages/x-components/src/components/x-component.mixin.ts

```diff
@@ -1,7 +1,6 @@
 import Vue, { ComponentOptions } from 'vue';
 import { XPlugin } from '../plugins/x-plugin';
 import { AnyXModule } from '../x-modules/x-modules.types';
-import { setXComponentXModuleName } from './x-component.utils';
 
 /**
  * Initializes a component as an X-Component:
```

Comment:
Does this function still do all the things written in the doc?

Reply:
Yes, it is still registering the module, and yes, setting the `xModule` property in the options object is the new way of flagging a component as an x-component.